mcs/build/profiles/testing_aot_full_interp.make

```diff
@@ -0,0 +1,35 @@
+#! -*- makefile -*-
+
+BOOTSTRAP_PROFILE = build
+
+BOOTSTRAP_MCS = MONO_PATH="$(topdir)/class/lib/$(BOOTSTRAP_PROFILE)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(INTERNAL_CSC)
+MCS = $(BOOTSTRAP_MCS)
+
+profile-check:
+	@:
+
+DEFAULT_REFERENCES = mscorlib
+
+PROFILE_MCS_FLAGS = \
+	-d:NET_1_1 \
+	-d:NET_2_0 \
+	-d:NET_2_1 \
+	-d:MOBILE,MOBILE_LEGACY \
+	-d:MOBILE_DYNAMIC \
+	-d:NET_3_5 \
+	-d:NET_4_0 \
+	-d:NET_4_5 \
+	-d:MONO \
+	-nowarn:1699 \
```

Comment:
This is missing
```
	-d:DISABLE_REMOTING \
	-d:DISABLE_COM \
```

It'd be much better to move the common setting to testing_aot_common.make and include it from these 3 config files (see monotouch_tv.make for example)

Reply:
You have disabled them in runtime https://github.com/mono/mono/pull/11307/files#diff-67e997bcfdac55191033d57a16d1408aR1232 and we don't really care about them for mobile profiles which you have enabled too